operator/v1/types_clustermanager.go

```diff
@@ -125,7 +125,7 @@ type RegistrationDriverHub struct {
 	// +kubebuilder:validation:Enum=csr;awsirsa
 	AuthType string `json:"authType,omitempty"`
 
-	// This represents the hub cluster ARN
+	// This represents the hub cluster ARN. Applicable to only awsirsa authentication type.
```

Comment:
Nit: Could you group the `Applicable to only awsirsa authentication type` fields together. 

To me the order should be: AuthType, AutoApprovedIdentities, HubClusterArn, Tags

Reply:
I can't resolve this thread. Please feel free to resolve it if you are able to. Thanks.